### ISSUE TITLE
A11y_AzureDataStudio_Notebook_MarkdownToolbar_ScreenReader : Incorrect reading order for the "Kernel" and "Attach to" controls for screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/azuredatastudio#11554]
**"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”**
GitHubTags:#A11y_AzureDataStudioJuly2020;#A11y_July27_2020_TestPass;#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#MAC;#Screenreader;#VoiceOver;#A11ySev2;#Benchmark;#MAS1.3.1;#MAS1.3.2;#FTP;

### Environment Details:
Application Name: Azure Data Studio
Application Version: 1.21.0-insider
Commit: eccf3cf
Date: 2020-07-24T09:28:31.172Z
VS Code: 1.48.0
Electron: 9.1.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0
Operating system: macOS Catalina (Version 10.15.6 (19G73)
Screen Reader: VoiceOver
MAS References: MAS1.3.1, MAS1.3.2

### Repro Steps:
1. Launch Azure Data Studio Insiders application.
2. Connect to server.
3. Navigate to File --> New Notebook.
4. Navigate upto + Cell button in notebook, hit enter & select markdown cell.
5. Start screen reader, navigate through the "Kernel" and "Attach to" controls and observe if the content is announced in proper sequence.

### Actual:
When screen reader users navigate to the "Kernel" and "Attach to" controls, after the name of the control, the "V" icon is announced then the dropdown control is announced to the screen reader users which is incorrect.

### Expected:
For screen reader users while navigation after the name of the control the dropdown control should be announced.

### User Impact:
If if the content present on the screen is not announced in proper sequence for the screen reader users then it will lead to confusion for the screen reader users.

### Attachment link for Reference:
[11554_A11y_AzureDataStudio_Notebook_MarkdownToolbar_ScreenReader Incorrect reading order for the "Kernel" and "Attach to" controls for screen reader users.zip](https://github.com/microsoft/azuredatastudio/files/4987257/11554_A11y_AzureDataStudio_Notebook_MarkdownToolbar_ScreenReader.Incorrect.reading.order.for.the.Kernel.and.Attach.to.controls.for.screen.reader.users.zip)

**Comment by SachinChavhan317** [comment]
Verified the bug on below version.
Version: 1.42.0-insider (Universal)

The issue is not reproducing, hence closing the bug. #closed;
Attachment:
![Mac2](https://user-images.githubusercontent.com/86241494/224310408-7527b299-e4d3-41a7-aeb1-e9a140ff6fd8.gif)
